src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const docPopulationArray = document.querySelectorAll('.population');
+
+const docTotal = document.querySelector('.total-population');
+
+const docAverage = document.querySelector('.average-population');
+
+const populationNumbers = [];
+
+docPopulationArray.forEach((item) => {
+  populationNumbers.push(parseInt(item.textContent.replaceAll(',', '')));
+});
+
+const totalPopulation = populationNumbers.reduce((total, curr) => total + curr);
+
+let averagePopulation;
+
+if (populationNumbers.length > 0) {
+  averagePopulation = totalPopulation / populationNumbers.length;
+}
```

Comment:
so what if population.length === 0? I think it should be zero.
Instead of using let you can just use ternary 
```suggestion
const averagePopulation = populationNumbers.length > 0 ?  totalPopulation / populationNumbers.length : 0;
```